### DISCUSSION TITLE
配置字段统一是否好一些

### DISCUSSION THREAD

**Discussion by lifenjoiner** [XTLS/Xray-core#359] [Ideas]
做了一些协议配置字段的对比：
![vx-conf](https://user-images.githubusercontent.com/19585474/110583307-73aae300-81a8-11eb-8e65-8e9a814c4b57.png)

对于箭头标示的，有一点建议：
1. `protocol`，值都用 URI 的 scheme；单独一个 shadowsocks，生成和解析的时候都要额外转换一下；ss，大家也都懂。
2. 都用`"settings": {"servers": [{`，`protocol` 已经设定了，无需特意 `vnext`，需要额外处理。

当然，这些是从`v2ray`，`v2fly`，`xray`一路演化过来的，不是 issue。感觉能统一的话，普通用户感觉不到，但是对开发是好的😂
只是，要改变当前正在使用的规则，还需要这些项目里都活跃的大佬来推动🌹🌹🌹

**Comment by rurirei** [comment]
agreed.

**Comment by AkinoKaede** [comment]
Related Issues:
v2fly/v2ray-core#190
